packages/server/lib/routes.ts

```diff
@@ -48,7 +49,10 @@ export const createCommonRoutes = ({
 
   router.get('/__cypress/iframes/*', (req, res) => {
     if (testingType === 'e2e') {
-      iframesController.e2e({ config, getSpec, getRemoteState }, req, res)
+      // ensure the remote state gets cleaned up from any previous tests/runs
+      remoteStates.reset()
```

Comment:
@mschile do we truly need to reset here and clientRoute? won't these iframes be loaded immediately after clientRoute is?